### DISCUSSION TITLE
Ian Wold | Book Club 12/2024: Team Metrics and KPIs

### DISCUSSION THREAD

**Discussion by giscus** [IanWold/ianwold.github.io#19] [Posts]
# Ian Wold | Book Club 12/2024: Team Metrics and KPIs

Ian Wold is a Software Engineer, Architect, and Team Leader in Minneapolis.

https://ian.wold.guru/Posts/book_club_12-2024.html

**Comment by andrewwillette** [comment]
KPI me daddy!

I've come to agree with integration tests good, unit tests for code coverage bad.
